### PR TITLE
SFX-94: Set up directories and build tools within View layer repository

### PR DESCRIPTION
**Pull request by JoeyDeol**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,37 @@ SFX View SDK containing pre-built web components for for creating e-commerce app
 ## Installation
 To clone this repo with submodules run:
 ```
-git clone --recursive https://github.com/groupby/sfx-logic
+git clone --recursive https://github.com/groupby/sfx-view
```
Comment (juliamlim) [inline README.md]: 🙏

File: packages/web-components/@sfx/sayt/package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@sfx/sayt",
+  "version": "0.0.0",
+  "description": "",
+  "main": "./src/index.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "../../../../scripts/build.sh",
+    "dev": "nodemon --watch src --exec npm run build"
+  },
```
Comment (juliamlim) [inline packages/web-components/@sfx/sayt/package.json]: Same feedback as the logic layer as per https://docs.npmjs.com/files/package.json#repository
```suggestion
  },
  "repository": {
  "type" : "git",
  "url" : "https://github.com/groupby/sfx-view.git",
  "directory": "packages/web-components/sayt"
},
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  "workspaces": [
+    "packages/web-components/@sfx/*"
+  ],
+  "scripts": {
+    "bundle": "webpack",
+    "start": "webpack-dev-server"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/groupby/sfx-view.git"
```
Comment (juliamlim) [inline package.json]: We should standardize this between the two packages
```suggestion
    "url": "https://github.com/groupby/sfx-view.git"
```

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'dist')
+  },
+
+  plugins: [new webpack.ProgressPlugin()],
+
+  module: {
+    rules: [{
+      test: /.(ts|tsx)?$/,
+      loader: 'ts-loader',
```
Comment (juliamlim) [inline webpack.config.js]: Also same feedback from logic, do we want to look into `awesome-typescript-loader`?

File: package.json (hunk 3 above)
Comment (JoeyDeol) [reply package.json]: Good call, I will update this.

File: packages/web-components/@sfx/sayt/package.json (hunk 2 above)
Comment (JoeyDeol) [reply packages/web-components/@sfx/sayt/package.json]: Oh cool, I didn't know about this. I will replicate this in both repos.

Comment (JoeyDeol) [reply packages/web-components/@sfx/sayt/package.json]: Updated.

File: webpack.config.js (hunk 4 above)
Comment (JoeyDeol) [reply webpack.config.js]: Updated.

File: package.json (hunk 3 above)
Comment (JoeyDeol) [reply package.json]: Updated.

File: README.md (hunk 1 above)
Comment (JoeyDeol) [reply README.md]: 🔥

**Pull request merged**